src/librustdoc/html/static/main.js

```diff
@@ -1710,7 +1706,7 @@ function defocusSearchBar() {
                     if (smallest === null) {
                         break;
                     }
-                    for (x = 0; x < arrays.length && ret.length < MAX_RESULTS; ++x) {
+                    for (x = 0; x < arrays_len && ret.length < MAX_RESULTS; ++x) {
```

Comment:
This is using the non-lexical variable from line 1700, right? JS is fun 😭.

Reply:
Yes. It's a syntax nightmare. ^^'